README.md

```diff
@@ -74,6 +75,10 @@ The above workflow (when manually triggered) will:
 
 ![image](https://user-images.githubusercontent.com/2510597/140506212-4938e44d-0662-4dc5-9fb1-c3f59fe075a6.png)
 
+
+*Note*: If you choose the "auto" option for semver version updates, the action will attempt to determine the new version number based on the commit messages. For this option to work, the repository must use the conventional commits standard.
```

Comment:
this looks good, but maybe we should dedicate a specific section to the auto option rather than a simple note in the readme